Xamarin.Forms.Platform.GTK/Renderers/TableViewRenderer.cs

```diff
@@ -34,6 +35,11 @@ protected override void Dispose(bool disposing)
 
 		protected override void OnElementChanged(ElementChangedEventArgs<TableView> e)
 		{
+			if (e.OldElement != null)
+			{
```

Comment:
Remove for one line block

Reply:
@PureWeen just like you, I prefer not to have them, but we have both styles in our code, and we shouldn't block a PR on this.

Reply:
Out of curiosity, and to take it into account in other PR, which is preferred?

Reply:
https://github.com/xamarin/Xamarin.Forms/blob/master/CONTRIBUTING.md#bug-fixes
> Follow the style used by the .NET Foundation, with two primary exceptions
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md

The two Primary Exceptions being tabs instead of spaces and not using private

And then from the .NET foundation

> We use Allman style braces, where each brace begins on a new line. A single line statement block can go without braces but the block must be properly indented on its own line and must not be nested in other statement blocks that use braces (See issue 381 for examples). One exception is that a using statement is permitted to be nested within another using statement by starting on the following line at the same indentation level, even if the nested using contains a controlled block.

I like to be consistent because then I can just hit CTRL E, D and rely on the editorconfig settings to ensure my changes match the coding style. In the past VS for Mac was a bit frustrating with indenting properly as well so I found it easier to just do document formats.

Reply:
Thanks @PureWeen!